frontend-new/src/components/Header.vue

```diff
@@ -1,17 +1,209 @@
-  </template>
+    <template v-if="announcements">
+        <Announcement v-for="(announcement, idx) in announcements" :key="idx" :announcement="announcement"/>
+    </template>
+    <header class="background-header">
+        <div class="inner-header flex items-center max-w-1200px mx-auto justify-between h-65px w-[calc(100%-40px)]">
+            <div class="logo-and-nav flex items-center">
+                <Popover class="relative">
+                    <PopoverButton v-slot="{ open }" class="flex mr-4">
+                        <icon-mdi-menu
+                            class="transition-transform text-[1.2em]"
+                            :class="open
+                            ? 'transform rotate-90'
+                            : ''"/>
+                    </PopoverButton>
+                    <transition
+                        enter-active-class="transition duration-200 ease-out"
+                        enter-from-class="translate-y-1 opacity-0"
+                        enter-to-class="translate-y-0 opacity-100"
+                        leave-active-class="transition duration-150 ease-in"
+                        leave-from-class="translate-y-0 opacity-100"
+                        leave-to-class="translate-y-1 opacity-0"
+                    >
+                        <PopoverPanel
+                            class="fixed md:absolute z-10 w-9/10 md:w-max background-header top-1/14 md:top-10 left-1/20 shadow1 rounded-md md:rounded-none md:rounded-bl-md md:rounded-r-md border-top-primary text-xs p-[20px]">
+                            <p class="text-base font-semibold color-primary mb-4">Hangar</p>
+                            <div class="grid grid-cols-2">
+                                <router-link
+                                    :to="{ name: 'index' }"
+                                    class="flex items-center rounded-md px-6 py-2"
+                                    hover="text-primary-100 bg-primary-50"
+                                >
+                                    <icon-mdi-home class="mr-3 text-[1.2em]"/>
+                                    Home
+                                </router-link>
+                                <router-link
+                                    :to="{ name: 'staff' }"
+                                    class="flex items-center rounded-md px-6 py-2"
+                                    hover="text-primary-100 bg-primary-50"
+                                >
+                                    <icon-mdi-account-group class="mr-3 text-[1.2em]"/>
+                                    Team
+                                </router-link>
+                            </div>
+
+                            <p class="text-base font-semibold color-primary mb-4 mt-10">More from Paper</p>
+                            <div class="grid grid-cols-2">
+                                <a
```

Comment:
pretty sure we want a loop here, you can use `<component is="icon-mdi-home" class=bla>`